client/app/styles/_base.scss

```diff
@@ -12,6 +12,10 @@
 // TODO: Remove this line once Service UI CONFIGURE button stops being added to Scope.
 .scope-wrapper .setup-nav-button { display: none; }
 
+a {
+  text-decoration: none;
```

Comment:
Note that this is scope-wide and brings it on par with the defaults of service-ui.